From roel.kluin@gmail.com  Tue Jun  2 12:16:43 2009
From: Roel Kluin <roel.kluin@gmail.com>
Date: Wed, 20 May 2009 02:36:29 +0200
Subject: Staging: otus: beyond ARRAY_SIZE of wd->ap.wds.encryMode
To: Greg KH <gregkh@suse.de>
Cc: Andrew Morton <akpm@linux-foundation.org>
Message-ID: <4A13508D.1070500@gmail.com>


Do not go beyond ARRAY_SIZE of wd->ap.wds.encryMode

Signed-off-by: Roel Kluin <roel.kluin@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/otus/80211core/coid.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/staging/otus/80211core/coid.c
+++ b/drivers/staging/otus/80211core/coid.c
@@ -2013,7 +2013,7 @@ u16_t zfiConfigWdsPort(zdev_t* dev, u8_t
 
     zmw_get_wlan_dev(dev);
 
-    if (wdsPortId > ZM_MAX_WDS_SUPPORT)
+    if (wdsPortId >= ZM_MAX_WDS_SUPPORT)
     {
         return ZM_ERR_WDS_PORT_ID;
     }
